modules/flake-parts/nixosConfigurations.stun-0.main.infra.holo.host/configuration.nix

```diff
@@ -0,0 +1,64 @@
+{
+  config,
+  inputs,
+  self,
+  pkgs,
+  ...
+}: let
+  # https://console.hetzner.cloud/projects/1982619/servers/47741841/overview
+  hostName = "stun-0";
+  domain = "main.infra.holo.host";
+  ipv4 = "37.27.39.142";
+  fqdn = "${config.networking.hostName}.${config.networking.domain}";
+in {
+  imports = [
+    inputs.disko.nixosModules.disko
+    inputs.srvos.nixosModules.server
+    inputs.srvos.nixosModules.mixins-terminfo
+    inputs.srvos.nixosModules.hardware-hetzner-cloud
+    self.nixosModules.hardware-hetzner-cloud-ccx
+
+    inputs.sops-nix.nixosModules.sops
+
+    self.nixosModules.holo-users
+    ../../nixos/shared.nix
+    ../../nixos/shared-nix-settings.nix
+
+    self.nixosModules.holochain-turn-server
+  ];
+
+  networking = {inherit hostName domain;};
+
+  hostName = ipv4;
+
+  nix.settings.max-jobs = 8;
+
+  nix.settings.substituters = [
+    "https://holochain-ci.cachix.org"
+  ];
+
+  nix.settings.trusted-public-keys = [
+    "holochain-ci.cachix.org-3:5IUSkZc0aoRS53rfkvH9Kid40NpyjwCMCzwRTXy+QN8="
+  ];
+
+  system.stateVersion = "23.11";
+
+  services.holochain-turn-server = {
+    enable = true;
+    url = fqdn;
+    address = ipv4;
+    listening-port = null;
+    nginx-http-port = 80;
```

Comment:
Is this correct? Wasn't the default 82? Will this conflict with the port 80 stun server?

Reply:
for this instance i coturn only listens on 443. this alleviates the need for the external reverse-proxy that intercepts the ACME challenge request.

Reply:
I think I'm comfortable with this for the foreseeable future. The performance benefits of a port 80 stun server (if there even are any) are probably minuscule enough that they wont materially matter to us.